acpi/src/fadt.rs

```diff
@@ -340,11 +338,172 @@ impl Fadt {
 }
 
 #[derive(Clone, Copy)]
-// TODO: methods for other flags
-pub struct Flags(u32);
+pub struct FixedFeatureFlags(u32);
 
-impl Flags {
+impl FixedFeatureFlags {
+    /// If true, an equivalent to the x86 [WBINVD](https://www.felixcloutier.com/x86/wbinvd) instruction is supported.
+    /// All caches will be flushed and invalidated upon completion of this instruction,
+    /// and memory coherency is properly maintained. The cache *SHALL* only contain what OSPM references or allows to be cached.
+    pub fn supports_equivalent_to_wbinvd(&self) -> bool {
+        self.0.get_bit(0)
+    }
+
+    /// If true, [WBINVD](https://www.felixcloutier.com/x86/wbinvd) properly flushes all caches, but memory coherency is not maintained.
```

Comment:
Comment is slightly incorrect: if `true`, caches are flushed and memory coherency *is* maintained, but caches are not guaranteed to be invalidated